articles/ai-services/speech-service/includes/release-notes/release-notes-stt.md

```diff
@@ -8,6 +8,11 @@ ms.author: eur
 
 ### September 2024 release
 
+
+#### [Real-time speech to text](../../how-to-recognize-speech.md) has released new models for follwoing languages with better quality. 
```

Comment:
```suggestion
#### Real-time speech to text 

[Real-time speech to text](../../how-to-recognize-speech.md) has released new models, with better quality, for the following languages. 

```
Fixing typo and moving sentence below the heading to follow the pattern of other headings in this article. @eric-urban - After this content is merged to main, you might want to reformat lines 13 and 14 as a table, such as the one in the "Pronunciation assessment" section below.